ui/apps/dev-server-ui/src/app/(dashboard)/stream/Stream.tsx

```diff
@@ -231,26 +232,35 @@ export default function Stream() {
 
   return (
     <div className="flex min-h-0 min-w-0 flex-col">
-      <div className="flex justify-end gap-1 px-5 py-2">
-        <Button
-          label={`${showInternalEvents ? 'Hide' : 'Show'} Internal Events`}
-          btnAction={() => setShowInternalEvents((prev) => !prev)}
-        />
-        <Button
-          label={freezeStream ? 'Resume Stream' : 'Freeze Stream'}
-          btnAction={() => setFreezeStream((prev) => !prev)}
-        />
-        <SendEventButton
-          label="Test Event"
-          data={JSON.stringify({
-            name: '',
-            data: {},
-            user: {},
-          })}
-        />
-      </div>
+      <Header
+        breadcrumb={[{ text: 'Stream' }]}
+        action={
+          <div className="flex justify-end gap-1 px-5 py-2">
+            <NewButton
+              kind="secondary"
+              appearance="outlined"
+              label={`${showInternalEvents ? 'Hide' : 'Show'} Internal Events`}
+              onClick={() => setShowInternalEvents((prev) => !prev)}
+            />
+            <NewButton
+              kind="secondary"
+              appearance="outlined"
+              label={freezeStream ? 'Resume Stream' : 'Freeze Stream'}
```

Comment:
I've long wanted to change this copy from "freeze." We don't use freeze anywhere, so I think leaning into "auto-refresh" as we will use elsewhere is better. 
```suggestion
              label={freezeStream ? 'Enable auto-refresh' : 'Disable auto-refresh'}
```